packages/boxel-ui/addon/components/input/validation-state/index.gts

```diff
@@ -10,10 +10,13 @@ interface ValidationStateInputArgs {
   disabled?: boolean;
   errorMessage?: string;
   helperText?: string;
-  id: string;
+  placeholder?: string;
+  id?: string;
   value: string;
   onInput?: (val: string) => void;
   onBlur?: (ev: Event) => void;
+  onKeyPress?: (ev: KeyboardEvent) => void;
+  onKeyDown?: (ev: KeyboardEvent) => void;
```

Comment:
Did you end of using `onKeyDown`? I only see `onKeyPress` used.

Reply:
Thanks, yeah I didn’t end up using it. I can clean up.